rtb/piano_di_qualifica_ver0.1.0.typ

```diff
@@ -0,0 +1,99 @@
+#import "/templates/template.typ": *
+
+#show: content => verbale(
+  data: "7 Dicembre 2024",
```

Comment:
[] dicembre in minuscolo (vedi NdP sez. 3.1.11)

N.B. ho notato che i documenti generali hanno la maiuscola, è da cambiare